code/script.js

```diff
@@ -1,47 +1,235 @@
+        .getElementById('false0')
+        .addEventListener('click', () => {showMessage('Sorry try again' , 'bot')})
+
+
+
+        currentQuestion++
+        console.log('Current question is now', currentQuestion)
+
+      }else if (subject === 'greetings') {
+          showMessage(`I choose ${subject}`, 'user')
+          showMessage(`Ok, greetings it is! translate: Hello, How are you in Swedish?`, 'bot')
+     
+          inputWrapper.innerHTML = `
+          
+          <button id="good">Hej hur mår du?</button>
+          <button id="false">Hallo hoe gaat het?</button>
+          <button id="false0">Var bor du?</button>
+        `
+        document 
+        .getElementById('good')
+        .addEventListener('click', () => {showMessage('That was the right answer', 'bot')
+
+        {inputWrapper.innerHTML = `
+        <button id="retry">Try again...</button>
+        <button id="end">End Programming</button>
+            `
+            
+
+            document
+            .getElementById('retry')
+            .addEventListener('click', () => {location.reload()})
+            document
+            .getElementById('end')
+            .addEventListener('click', () => {showMessage('Congratiolations on completing the program!' , 'bot')})
+       }
+      })
+
+        document
+        .getElementById('false')
+        .addEventListener('click', () => {showMessage('Sorry try again' , 'bot')})
+        document
+        .getElementById('false0')
+        .addEventListener('click', () => {showMessage('Sorry try again' , 'bot')})
+
+        currentQuestion++
+        console.log('Current question is now', currentQuestion)
+
+
+      }else {
+        showMessage(`I choose ${subject}`, 'user')
+        showMessage(`Ok, we will talk about jobs. translate this: My father is a judge`, 'bot')
+
+          inputWrapper.innerHTML = `
+    
+          <button id="false">Min far är läkara</button>
+          <button id="false0">Mijn vader is een rechter?</button>
+          <button id="good">Min far är domare</button>
+        
+         `
+         document
+         .getElementById('false0')
+         .addEventListener('click', () => {showMessage('Sorry try again' , 'bot')})
+          document
+          .getElementById('false')
+          .addEventListener('click', () => {showMessage('Sorry try again', 'bot')
+          })
+         document
+         .getElementById('good')
+         .addEventListener('click', () => {showMessage('That was the right answer', 'bot')
+
+         {inputWrapper.innerHTML = `
+          <button id="retry">Try again...</button>
+          <button id="end">End Programming</button>
+              `
+              
+
+              document
+              .getElementById('retry')
+              .addEventListener('click', () => {location.reload()})
+              document
+              .getElementById('end')
+              .addEventListener('click', () => {showMessage('Congratulations on completing the program!', 'bot')
```

Comment:
When most of the questions, answers and final outcomes are within the single `askForType` function, it has the downside of creating a lot of repetition. To reduce that it would probably be worth looking into breaking the stages of answers/questions out into their own functions to be called upon only when needed.